Tasks/SqlAzureDacpacDeploymentV1/DeploySqlAzure.ps1

```diff
@@ -58,6 +58,15 @@ function ThrowIfMultipleFilesOrNoFilePresent($files, $pattern)
     }
 }
 
+function EscapeSpecialChars
+{
+    param(
+        [string]$str
+    )
+
+    return $str.Replace('`', '``').Replace('$', '`$')
```

Comment:
Can you verify once for single quotes too in the password?

Reply:
Already verified $Ro`s"'2016 .It also contains single quote .

Reply:
:)